src/Assignments/SimpleProblemPage.tsx

```diff
@@ -119,6 +119,7 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
             }
             
             if (actualVersionsRemaining > 0) {
+                // TODO do we need to check for start times?
```

Comment:
What do you mean? right now the backend throws an error if you try to access it early so you would never get here

Reply:
when the start assessment bug was found - I couldn't tell where I'd gone wrong. This was one possibility.